Traceback (most recent call last):
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 211, in <module>
    episode_rewards = mini_batch_train(
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 164, in mini_batch_train
    action = agent.get_action(state)
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 81, in get_action
    qvals = self.model.forward(state)
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 53, in forward
    x = x.view(128, 390)
RuntimeError: shape '[128, 390]' is invalid for input of size 390
/usr/lib/python3/dist-packages/apport/report.py:13: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import fnmatch, glob, traceback, errno, sys, atexit, imp, stat
Traceback (most recent call last):
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 211, in <module>
    episode_rewards = mini_batch_train(
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 164, in mini_batch_train
    action = agent.get_action(state)
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 81, in get_action
    qvals = self.model.forward(state)
  File "/home/navin/projects/M2P/Re_M2P/agent5_ddqn.py", line 53, in forward
    x = x.view(128, 390)
RuntimeError: shape '[128, 390]' is invalid for input of size 390
